src/jit/lower.cpp

```diff
@@ -2437,7 +2399,7 @@ void Lowering::LowerCompare(GenTree* cmp)
         GenTreeIntCon* op2      = cmp->gtGetOp2()->AsIntCon();
         ssize_t        op2Value = op2->IconValue();
 
-        if (IsContainableMemoryOp(op1) && varTypeIsSmall(op1Type) && genTypeCanRepresentValue(op1Type, op2Value))
```

Comment:
Wouldn't have been easier to just change `Lowering::IsContainableMemoryOp` to call `m_lsra->isContainableMemoryOp`?

Reply:
Good suggestion. Done.